modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/folders/SuggestedFoldersPaywall.kt

```diff
@@ -0,0 +1,167 @@
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun SuggestedFoldersPaywall(
+    onUseTheseFolders: () -> Unit,
+    onMaybeLater: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .wrapContentSize()
+            .padding(horizontal = 16.dp)
+            .padding(top = 8.dp, bottom = 16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        val isPortrait = LocalConfiguration.current.orientation == Configuration.ORIENTATION_PORTRAIT
+
+        val animatedPadding by animateDpAsState(
+            targetValue = if (isPortrait) 32.dp else 8.dp,
+        )
+
+        Icon(
+            painter = painterResource(IR.drawable.ic_swipe),
+            contentDescription = null,
+            tint = MaterialTheme.theme.colors.primaryUi05,
+            modifier = Modifier.clearAndSetSemantics {},
+        )
+
+        AnimatedVisibility(isPortrait) {
+            Folders(
+                podcastUuids = mockedPodcastsUuids,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = animatedPadding)
+                    .align(Alignment.CenterHorizontally),
+            )
+        }
+
+        TextH30(
+            text = stringResource(LR.string.suggested_folders_paywall_tittle),
+            fontWeight = FontWeight.W600,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.padding(bottom = 12.dp),
+        )
+
+        TextH50(
+            text = stringResource(LR.string.suggested_folders_paywall_subtitle),
+            color = MaterialTheme.theme.colors.primaryText02,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.padding(bottom = 12.dp),
+        )
+
+        RowButton(
+            text = stringResource(LR.string.suggested_folders_use_these_folders_button),
+            modifier = Modifier.padding(bottom = 16.dp),
+            textColor = MaterialTheme.theme.colors.primaryInteractive02,
+            fontSize = 18.sp,
+            fontWeight = FontWeight.W600,
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = MaterialTheme.theme.colors.primaryInteractive01,
+            ),
+            includePadding = false,
+            onClick = onUseTheseFolders,
+        )
+
+        RowOutlinedButton(
+            text = stringResource(id = LR.string.maybe_later),
+            modifier = Modifier.padding(bottom = 16.dp),
+            onClick = onMaybeLater,
+            includePadding = false,
+            colors = ButtonDefaults.outlinedButtonColors(contentColor = MaterialTheme.theme.colors.primaryIcon01, backgroundColor = Color.Transparent),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.W600,
+        )
+    }
+}
+
+@Composable
+private fun Folders(podcastUuids: List<String>, modifier: Modifier = Modifier) {
+    val folders = 3
+
+    LazyVerticalGrid(
```

Comment:
I might be missing something by why use `LazyVerticalGrid` over a `Row`?

Reply:
I used this grid because I was not sure if we will display only 3 items or more

Reply:
Is it still unclear or do you know if there will be only items?

Reply:
I was able to simplify this by replacing the LazyGrid for LazyRow [d9a80c0](https://github.com/Automattic/pocket-casts-android/pull/3554/commits/d9a80c0b1606d32af98ee3b99736c790f22ad6b7)